tests/utils.js

```diff
@@ -21,6 +21,12 @@ export function getCss(style) {
       .join('')
 }
 
+export function getCssFromSheet(sheet) {
+  return [...sheet.renderer.getRules()]
```

Comment:
Is babel using Array.from? I would simply list the required polyfils in the readme